includes/class-wc-stripe-order-handler.php

```diff
@@ -333,6 +334,50 @@ public function cancel_payment( $order_id ) {
 			do_action( 'woocommerce_stripe_process_manual_cancel', $order );
 		}
 	}
+
+	/**
+	 * Filter. Adds additional meta data to Tracks events.
+	 * Note that this filter is only called if WC_Site_Tracking::is_tracking_enabled.
+	 *
+	 * @since 4.5.1
+	 * @param array Properties to be appended to.
+	 * @param string Event name, e.g. orders_edit_status_change.
+	 */
+	public function woocommerce_tracks_event_properties( $properties, $prefixed_event_name ) {
+		// Not the desired event? Bail.
+		if ( 'wcadmin_orders_edit_status_change' != $prefixed_event_name ) {
```

Comment:
I only got this event to be tracked when an order was updated using the admin UI. Is that the expected behavior? Or should it be logged when the order is completed automatically too (e.g. virtual products)?

Reply:
It should be logged either way. I will re-test that.

Reply:
Hmmm. Not happening for me either when a product is virtual+downloadable.

Reply:
OK. Tracked it down. WooCommerce Tracking is only supported for admin views, so even though we are hooking the woocommerce_order_status_changed action, we only do so when is_admin() is true.

There is no front side tracking supported yet at all, so orders that automatically progress to completed will never fire a Tracks event.

Reply:
Gotcha. Based on https://github.com/woocommerce/woocommerce/issues/25069#issuecomment-670213275 I'll go ahead and approve this, as we'll take care of the orders that are complete automatically in https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1275.